prac_06/programming_language.py

```diff
@@ -0,0 +1,19 @@
+"""Programming Language Class"""
+
+
+class ProgrammingLanguage:
+    def __init__(self, name, typing, reflection, year):
+        self.name = name
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+
+    def is_dynamic(self):
+        if self.typing == 'Dynamic':
+            return True
+        else:
+            return False
```

Comment:
This could be condensed into a single return. For example:
return True if self.typing.lower() == "dynamic" else False